src/repertoire/schema/version.py

```diff
@@ -0,0 +1,25 @@
+"""SQLAlchemy schema for TAP_SCHEMA version table."""
+
+from __future__ import annotations
+
+from datetime import datetime
+
+from sqlalchemy import TIMESTAMP, String, func
+from sqlalchemy.orm import Mapped, mapped_column
+
+from .base import SchemaBase
+
+__all__ = ["TAPSchemaVersion"]
+
+
+class TAPSchemaVersion(SchemaBase):
+    """Table holding TAP_SCHEMA version information."""
+
+    __tablename__ = "version11"
```

Comment:
If we don't want to hard code the table name and schema name I'm not sure how we'd do it other than going back to manually creating the table with Table() in _record_version() instead of using the ORM model or generating the model via the factory.